cmd/neofs-ir/defaults.go

```diff
@@ -94,6 +94,7 @@ func defaultConfiguration(cfg *viper.Viper) {
 	cfg.SetDefault("emit.mint.cache_size", 1000)
 	cfg.SetDefault("emit.mint.threshold", 1)
 	cfg.SetDefault("emit.mint.value", 20000000) // 0.2 Fixed8
+	cfg.SetDefault("emit.gas.balance_threshold", 10)
```

Comment:
need help with naming and default value

Reply:
As for naming it's fine. We can reuse it later to make a limit on all side chain GAS transfer operations, so it is good.
As for default value, let's keep it zero. We can't provide any reasonable default value there.

Reply:
ok, changed to zero